website/docs/r/google_service_account_key.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "google"
+page_title: "Google: google_service_accout_key"
+sidebar_current: "docs-google-service-account-key"
+description: |-
+  Allows management of a Google Cloud Platform service account Key Pair
+---
+
+# google\_service\_account\_key
+
+Allows management of a key, and must be created or imported for use with
```

Comment:
It would be nice to have a bit more text on what a key is. Maybe something like:
```
Creates and manages service account key-pairs, which allow the user to establish identity of a service account outside of GCP. For more information, see [the official documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) and [API](https://cloud.google.com/iam/reference/rest/v1/projects.serviceAccounts.keys).
```